src/jit_channel/channel_manager.rs

```diff
@@ -224,7 +224,7 @@ impl OutboundJITChannelState {
 				compute_opening_fee(
 					expected_outbound_amount_msat,
 					opening_fee_params.min_fee_msat,
-					opening_fee_params.proportional,
+					opening_fee_params.proportional.into(),
```

Comment:
should we just store proportional as a u64 in [Raw]OpeningFeeParams and avoid these conversions?

Reply:
yeah i thought we had talked about this before. okay sounds good